.azure-pipelines/steps/download-samples.yml

```diff
@@ -4,17 +4,17 @@ parameters:
     default: ''
 
 steps:
-- task: DownloadPipelineArtifact@2
-  displayName: Download standalone samples
-  inputs:
+- template: download-artifact.yml
+  parameters:
     artifact: samples-standalone
     path: $(outputDir)/bin
-  retryCountOnTaskFailure: 5
+    displayName: Download standalone samples
+    retryCountOnTaskFailure: 5
```

Comment:
IMO we should try to reduce the amount of code wherever we can, so I think we can just ditch these 🤷‍♂️  The `5` was arbitrary anyway😄 
```suggestion
```